webapi/src/main/twirl/org/knora/webapi/queries/sparql/v2/isPropertyUsed.scala.txt

```diff
@@ -3,21 +3,18 @@
  * SPDX-License-Identifier: Apache-2.0
  *@
 
-@import org.knora.webapi.IRI
-
 @*
- * Checks whether a property is used (i.e., is the property of any statements).
+ * This query is asking whether there are any instances of a class or subclass that have a specific property.
+ * I.e. checks whether a property is used, for example is the property of any statements.
```

Comment:
I don't understand "for example is the property of any statements"

Reply:
I have tried to rephrase the old documentation to be more clear. Guess I have failed. Would it be clearer to write 
`or example is the predicate of any statements.`?

Reply:
how about "for example whether it is the predicate of any statements"?

Reply:
done: 53cb70dbd0bffe955b2e3883229b118c53ae3f1d